src/contentScript.js

```diff
@@ -67,8 +67,12 @@ function injectSimmer() {
     const script = document.createElement('script');
     script.setAttribute('id', 'fathom-simmer');
     script.setAttribute('src', browser.extension.getURL('simmer.js'));
-    const head = document.head || document.getElementsByTagName('head')[0] || document.documentElement;
-    head.insertBefore(script, head.lastChild);
+    script.setAttribute('onload', `
+        window.Simmer = window.Simmer.configure({
+            depth: 25
+        });
+    `);
```

Comment:
note this has to be an inline script; webextensions can't modify the window object directly.

Reply:
Did you try [the new `new Simmer(window, { /* some custom configuration */ })` syntax](https://www.npmjs.com/package/simmerjs)? I took a quick spin, but my first try didn't work.

Reply:
Take a look at https://github.com/erikrose/fathom-fox/commit/bdfb823dbb7bcb5776128dd0f4c4e7d86c825ea3 (tip of my repo's "simmer-down" branch). I seem to have got it working without the onload() handler. Does it look good to you?

Reply:
(I lean toward this because it removes the variable of needing an onload() script to fire; the config will always happen in time to make a selector.)

Reply:
`Simmer.configure({..})` works, however that would create a new simmer object on each call instead of using a pre-configured object so i wanted to avoid doing that.  if it's a light object to create and configure doing that at invocation seems reasonable to me.

Reply:
It's pretty light: basically a few `...` object expansions and an `===` check, along with returning a closure. If it becomes a problem, we can go back.